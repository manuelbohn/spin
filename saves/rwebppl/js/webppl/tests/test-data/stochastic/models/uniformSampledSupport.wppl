// Expected values estimated using forward sampling.
var model = function() {
  var upper = uniform(0.9, 1.0);
  var x = uniform(0, upper);
  // Ensure sampled values are within support. (#912)
  assert.ok(0 <= x && x <= upper);
  return x;
};
