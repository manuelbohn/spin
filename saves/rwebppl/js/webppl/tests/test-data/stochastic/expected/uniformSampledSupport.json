{
  "mean": 0.475,
  "std": 0.275
}
